crates/autopilot/src/arguments.rs

```diff
@@ -359,13 +359,11 @@ pub struct FeePolicy {
     /// - Volume based:
     /// volume:0.1
     #[clap(long, env, default_value = "surplus:0.0:0.9")]
-    pub fee_policy_kind: FeePolicyKind,
+    pub in_market_fee_policy_kind: FeePolicyKind,
 
-    /// Should protocol fees be collected or skipped for orders whose
-    /// limit price at order creation time suggests they can be immediately
-    /// filled.
-    #[clap(long, env, action = clap::ArgAction::Set, default_value = "true")]
-    pub fee_policy_skip_market_orders: bool,
+    /// Type of fee policy to use for out of market orders.
+    #[clap(long, env, default_value = "surplus:0.0:0.9")]
+    pub out_of_market_fee_policy_kind: FeePolicyKind,
```

Comment:
Adding it like this seems a bit sketchy. Could we already handle applying multiple fees to the same order?
If so we should be able to make the in-market condition part of the policy itself and simply configure a list like:
`surplus:0.0:0.9:market|surplus:0.0:0.8:limit`

Reply:
With the proposed approach, it is not really clear how to provide default values in a case where either market or limit protocol fee is only provided.

Reply:
I don't quite understand. Possible options could be to use `any` if a fee should apply to both limit and market orders or an empty list if no fee should apply. Would that not resolve that issue?

Reply:
The current method enables the utilization of default values for fees when one of them is unspecified. For instance, if only a limit fee is provided, a default market fee of 0.0 is applied for in-market orders, as indicated by the argument declaration. If the argument's type were to be changed to a `Vec`, and only the limit fee configuration is provided, the question arises as to which fee would then be applied for the in-market order. Would this necessitate hardcoding the fee?

Reply:
> a default market fee of 0.0 is applied for in-market orders

Is that not equivalent to not having a fee which would be in line with the array idea? I'm lost.

Reply:
My bad, I didn't notice that we are sending vector.

Reply:
Updated